spmd/requirements.txt

```diff
@@ -1,2 +1,10 @@
 # Python dependencies required for release
-torch >= 1.13.0.dev0
+--pre --extra-index-url https://download.pytorch.org/whl/nightly/cu113 sys_platform == "linux"
```

Comment:
Unfortunately https://pytorch.org/get-started/locally/ now only provides `cu102`, `cu116`, and `cu117`.

Reply:
it actually have more than these three, that's how our CI works before this change. I think for AWS node it's very tricky, I just tested even without this PR, with a clean env and do instruction exactly as https://pytorch.org/get-started/locally/  but it still can't install correct nightly (installed a stable version instead..)

Reply:
Interesting, so it's actually a problem AWS node. 

My date point:
- clean env
- Python=3.10
- nightly/cu113 failed
- nightly/cu116 succeeded

If we only need to make sure this PR doesn't break CI, then we can keep `cu113` in the file.

Reply:
You can ssh into the CI machine if you want to really verify that.

Reply:
@XilunWu Have you tried command here: https://pytorch.org/get-started/locally/?

Reply:
@fduwjj Yes they have `cu116` which works on AWS. I think this is a minor issue so I'll withdraw the change request.

Reply:
Thanks for discussing this, I did some experiment and it looks like AWS cluster can't install cu113, but CI can only install cu113, so I updated the PR to include both cu113 and cu116, so that both environment could install their corresponding version